package-lock.json

```diff
@@ -71,6 +71,28 @@
         "vscode": "^1.67.0"
       }
     },
+    "examples/hello-world": {
```

Comment:
This should not happen. Failed the CLI test, or is the clean-up after test disabled? 
Afaik I set it up to clean up even on test failure.